Jenkinsfile.deploy

```diff
@@ -0,0 +1,16 @@
+@Library(value='PipelineLibs@0.6.1', changelog=false) _
```

Comment:
Might want to use `0.6.3` here to mirror the library version in the build file.

Reply:
> Might want to use `0.6.3` here to mirror the library version in the build file.

I'm actually thinking it's going to be the next version (e.g. `0.7.0`) that we'll be using, so I left as Dan had it. The next library version has a new parameter that's important for the server to work.